tests/integration/targets/aci_interface_blacklist/tasks/main.yml

```diff
@@ -368,7 +368,7 @@
       <<: *aci_info
       <<: *test_vars_fex
       state: absent
-    check_mode: yes
+    check_mode: true
     register: enable_fex_int_check
 
   - name: Fex - Verify that outofsvc_interface_blacklist remains created after check mode
```

Comment:
Please check test file of aci_interface_policy_cdp for admin_state: no

Reply:
resolved